xwiki-rendering-api/src/main/java/org/xwiki/rendering/block/UnchangedContentBlock.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.rendering.block;
+
+import java.util.List;
+
+import org.xwiki.rendering.listener.MetaData;
+import org.xwiki.stability.Unstable;
+
+/**
+ * This block represents a part of a macro content which is not transformed during the macro execution.
+ *
+ * @version $Id$
+ * @since 10.9RC1
+ */
+@Unstable
+public class UnchangedContentBlock extends MetaDataBlock
```

Comment:
Wasn't the plan to also indicate if the content come from the macro content or some macro parameter ?

Reply:
Yes for me it was again another step.

Reply:
OK.